app/styles/components/_panel.scss

```diff
@@ -86,6 +86,19 @@ $panel-content-max-width: 500px;
   background-color: $Theme-color-white;
 }
 
+.Panel--solidLightBlue {
+  background-color: $Theme-color-light-blue;
+}
+
+.Panel--lightBlue {
+  color: $Theme-color-white;
+
+  .Panel-titleImage {
```

Comment:
I understand what you're saying, I merely replicated code that was already in place for Panel--blue, the need for this arose from having changed the bgColor of the panel; text and titleImg color would then change from thematic dark to a light color.
I should create a modifier class for Panel-titleImage something like Panel-titleImage--light/white for specials cases where titleImage needs to be white, and then manually apply that class in the markup.
Then refactor both of these usages on lines 73(Panel--blue) and line 93(Panel--lightBlue), right?